pool/app/pool_database/migrations/migration_202501091612.ml

```diff
@@ -0,0 +1,229 @@
+let create_mapping_table ~table_suffix ~column_name ~fk_constraint =
+  Database.Migration.Step.create
+    ~label:"add unique combination to pool_queue_jobs_mapping"
+    [%string
+      {sql|
+      CREATE TABLE pool_queue_job_%{table_suffix} (
```

Comment:
As discussed, add "_map" to name, otherwise might be confusing with "_history"

```suggestion
      CREATE TABLE pool_queue_job_map_%{table_suffix} (
```